kubernetes-manifests/accounts-db.yaml

```diff
@@ -33,8 +33,9 @@ spec:
         tier: db
     spec:
       containers:
-      - name: accounts-db
+      - name: postgres
         image: gcr.io/bank-of-anthos/accounts-db:v0.1.1
+        imagePullPolicy: "IfNotPresent"
```

Comment:
Was there a reason for adding this?

Reply:
It's in ledger-db.yaml.  Wanted to make things consistent.

Reply:
Apparently [IfNotPresent is the default when imagePullPolicy is not set](https://kubernetes.io/docs/concepts/containers/images/#updating-images)

Removing.